pkg/server/server.go

```diff
@@ -437,8 +444,13 @@ func (s *Server) handleHelmRenderManifests(user *auth.User, w http.ResponseWrite
 func (s *Server) handleHelmInstall(user *auth.User, w http.ResponseWriter, r *http.Request) {
 	params := r.URL.Query()
 	ns := params.Get("ns")
+	fValues := r.FormValue("values")
```

Comment:
The `f-` prefix typically indicates a `flag` in this repo.  Consider a rename to `formValues`.